test/e2e/workflow/deployment_test.go

```diff
@@ -198,3 +256,33 @@ func ignoreInitialKubeMacPoolRestart() {
 	By("Ignoring initial KubeMacPool restart")
 	time.Sleep(10 * time.Second)
 }
+
+func retrieveRange() (string, string) {
+	configMap := &v1.ConfigMap{}
+	var (
+		rangeStart, rangeEnd string
+	)
+	Eventually(func() error {
```

Comment:
Please add comment similar to https://github.com/kubevirt/cluster-network-addons-operator/pull/148/files#diff-fb59c5d6532b4a1311ba03eb02146c67R252